app/src/main/shared/Helpers.ts

```diff
@@ -3,7 +3,7 @@ export function doNothing() {
 }
 
 export function encodeFilename(filename: string) {
-    return filename.replace("/", ":");
+    return filename.replace(/\//g, ":");
```

Comment:
I find this regex a little unreadable. Perhaps it would be simpler to use the Regex constructor rather than a regex literal, like so: `new RegExp("/", "g")`

Reply:
done :)